src/app/_components/GovernanceDocsCard.tsx

```diff
@@ -0,0 +1,37 @@
+import { ArrowUpRight } from '@phosphor-icons/react/dist/ssr'
+
+import { CustomLink } from '@/components/CustomLink'
+import { type HeadingProps, Heading } from '@/components/Heading'
+
+type GovernanceDocsCardProps = {
+  heading: HeadingProps
+  cta: {
+    href: string
+    text: string
+  }
+  children: React.ReactNode
+}
+
+export function GovernanceDocsCard({
```

Comment:
I think we can use the `HomeExploreSectionCard` - No need to change the name for now.